app/views/votes/new.html.erb

```diff
@@ -1,19 +1,18 @@
-
 <div class="row">
   <div class="col s6 offset-s3">
     <h1>Vote for <%= @project.name %></h1>
     <%= form_for([:project, @vote]) do |f| %>
       <div class="input-field col s12">
         <%= f.select(:representation, [1, 2, 3, 4, 5], selected: 3) %>
-        <%= f.label :representation %>
+        <%= f.label "Does this project represent the student community?" %>
```

Comment:
I'm not sure if this will make a difference, but I think this should be: 
`f.label :representation, "Does this project represent the student community?"`